src/main/java/at/hannibal2/skyhanni/api/GetFromSackAPI.kt

```diff
@@ -222,12 +223,26 @@ object GetFromSackAPI {
     }
 
     @SubscribeEvent
-    fun onRepoReload(event: RepositoryReloadEvent) {
-        sackListInternalNames = event.getConstant<SacksJson>("Sacks").sackItems.toSet()
+    fun onNeuRepoReload(event: NeuRepositoryReloadEvent) {
+        val data = event.getConstant("sacks") ?: ErrorManager.skyHanniError("NEU sacks data is null.")
+        try {
+            val sacksData = ConfigManager.gson.fromJson<NeuSacksJson>(data).sacks
+            val uniqueSackItems = mutableSetOf<NEUInternalName>()
+
+            sacksData.values.forEach { sackInfo ->
+                sackInfo.contents.forEach { content ->
+                    uniqueSackItems.add(content)
+                }
+            }
+
+            sackListInternalNames = uniqueSackItems.map { it.asString() }.toSet()
+            sackListNames = uniqueSackItems.map { it.itemNameWithoutColor.uppercase() }.toSet()
```

Comment:
Instead of having `sackListInternalNames` and `sackListNames` separate, how about using one map instead?

Maybe also use a list/map of item names split by _ instead of empty lines, so we don't need to do this here
 https://github.com/hannibal002/SkyHanni/blob/9d696771b9c608c3afee2811dbebb1a3869b9f8f/src/main/java/at/hannibal2/skyhanni/features/commands/tabcomplete/GetFromSacksTabComplete.kt#L19

Reply:
we need to set. The code would else not work. Since the behavior of Names and InternalNames handling is completely different do to the underline Replace